rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerEventReceiver.java

```diff
@@ -101,6 +102,11 @@ private void openAndReceive(final SchedulerEventListener eventListener, boolean
         socket = client.create();
         EventNotificationHandler notificationHandler = new EventNotificationHandler(eventListener);
         socket.on(Event.MESSAGE, notificationHandler);
+        socket.on(Event.CLOSE, new Function() {
+            public void on(Object t) {
+                SchedulerEventReceiver.logger.info("#### websocket connection is closed ####");
```

Comment:
I think these logs should be put in debug mode rather than in info. Besides that:
- the ### are not necessary
- the log message should print some information about the connection which is being closed, to make the log message useful.